src/components/pill-selector-dropdown/PillSelectorDropdown.js

```diff
@@ -243,16 +243,16 @@ class PillSelectorDropdown extends React.Component<Props, State> {
         } = this.props;
 
         return (
-            <SelectorDropdown
-                className={classNames('pill-selector-wrapper', className)}
-                dividerIndex={dividerIndex}
-                onEnter={this.handleEnter}
-                onSelect={this.handleSelect}
-                overlayTitle={overlayTitle}
-                scrollBoundarySelector={dropdownScrollBoundarySelector}
-                shouldSetActiveItemOnOpen={shouldSetActiveItemOnOpen}
-                selector={
-                    <Label text={label}>
+            <Label text={label}>
```

Comment:
It looks like Label uses Tooltip, which uses TetherComponent. Will there be any impact on styling or behavior if the DOM elements of SelectorDropdown are now attached to the body rather than their parent?

Reply:
I think it still works ok. Here's an example with the field disabled:

<img width="415" alt="Screen Shot 2019-10-22 at 3 02 10 PM" src="https://user-images.githubusercontent.com/1280912/67337975-f7453080-f4dc-11e9-8449-223902e7b193.png">

Reply:
Looking again, Label only wraps its children with a Tooltip if the `tooltip` prop is defined (go figure), which isn’t the case here. Sorry for the false alarm!